### ISSUE TITLE
Switch to Command Mode and return to Idle Mode, will make suggestion on click to go any of Command Mode.

### ISSUE THREAD

**Issue by angrezichatterbox** [scribe-org/Scribe-Android#264]
### Terms

- [X] I have searched all [open bug reports](https://github.com/scribe-org/Scribe-Android/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
- [X] I agree to follow Scribe-Android's [Code of Conduct](https://github.com/scribe-org/Scribe-Android/blob/main/.github/CODE_OF_CONDUCT.md)

### Behavior

The user on click of Scribe Key on the keyboard and gets back directly to idle mode would result in the suggestion key to navigate the user to the common mode where we can type on the command bar. 

<img src="https://github.com/user-attachments/assets/e8d3449e-c1e4-412c-9398-fa92581d62b6" alt="Alt Text" width="300" />

### Device type

Medium Phone API 35

**Comment by andrewtavis** [comment]
Thanks for making this, @angrezichatterbox!

**Comment by Linfye** [comment]
Yes, this bug has existed for quite some time. Thank you for bringing it up. 😊
